### PR TITLE
DW-954 creation of contact policy layout

### PR DESCRIPTION
**Pull request by GusBaamonde**
layout-contact-policy02

https://cdn.fromdoppler.com/doppler-ui-library/build1227/contact-policy.html

#Checklist

- [X] Check if there is a new color.
- [X] Create the color variable in the _colors.scss file.
- [X] Follow the color nomenclature.
- [X] Create the color class with its variable.
- [X] Create the color component in the html, show the name, color class and its exadecimal.

### CODE REVIEW

File: assets/templates/contact-policy.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+                    >Limitar la cantidad maxima de envios a un mismo
+                    Suscriptor</label
+                  >
+                </li>
+                <li class="field-item">
+                  <span>No enviar mas de 20:</span>
+                  <input
+                    type="number"
+                    name="number"
+                    id="email"
```
Comment (cbernat) [inline assets/templates/contact-policy.html:135]:
```suggestion
                    id="emails"
```

File: assets/templates/contact-policy.html [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+                    placeholder=""
+                  />
+                  <span>Emails en un periodo de</span>
+                  <input
+                    type="number"
+                    name="number"
+                    id="email"
```
Comment (cbernat) [inline assets/templates/contact-policy.html:142]:
```suggestion
                    id="days"
```

**Comment by cbernat** [comment]
971x969 there is enough space for it to render correctly... 
image

File: assets/templates/contact-policy.html [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+                    placeholder=""
+                  />
+                  <span>Días</span>
+                </li>
+                <li class="field-item"><hr /></li>
+                <li class="fiel-item">
```
Comment (cbernat) [inline assets/templates/contact-policy.html:148]:
```suggestion
                <li class="field-item">
```

File: assets/templates/contact-policy.html [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+        <span class="arrow"></span>
+      </div>
+    </header>
+    <section class="dp-container">
+      <div class="dp-rowflex">
+        <div class="col-md-6 col-sm-12 m-b-24">
+          <form class="dp-contact-policy-form">
+            <fieldset>
+              <legend>Política de contacto</legend>
+              <ul class="field-group">
```
Comment (cbernat) [inline assets/templates/contact-policy.html:112]: I'm not so sure about rendering all the layout as an ul, this way we do not take advantage of the grid, it shows in smaller resolutions.

**Comment by cbernat** [comment]
https://cdn.fromdoppler.com/doppler-ui-library/build1226/contact-policy.html

**Comment by cbernat** [comment]
Very nice! 
image

File: assets/templates/contact-policy.html [hunk 5]
```diff
@@ -0,0 +1,197 @@
...
+                      <input
+                        type="number"
+                        name="number"
+                        id="correos electrónicos"
+                        placeholder=""
+                      />
+                    </div>
+
+                    <div>
+                      <span>Emails en un período de</span>
```
Comment (cbernat) [inline assets/templates/contact-policy.html:143]: Just a very minor detail, the word Emails belongs to the input above. 
No enviar mas de 20: ___ Emails
en un período de ___ días

File: assets/templates/contact-policy.html [hunk 6]
```diff
@@ -0,0 +1,197 @@
...
+                      <input
+                        type="number"
+                        name="number"
+                        id="dias"
+                        placeholder=""
+                      />
+                      <span>Días</span>
```
Comment (cbernat) [inline assets/templates/contact-policy.html:150]:
```suggestion
                      <span>días</span>
```

**Comment by GusBaamonde** [comment]
https://cdn.fromdoppler.com/doppler-ui-library/build1228/contact-policy.html

**Pull request merged**